tinytga/src/lib.rs

```diff
@@ -1,4 +1,80 @@
-//! No-std compatible TGA parser designed for embedded systems, but usable anywhere
+//! A small TGA parser designed for embedded, no-std environments but usable anywhere. Beyond
+//! parsing the image header, no other allocations are made. A reference to the input image data is
+//! kept and slices are returned from it.
+//!
+//! Call `Tga.into_iter()` to get an iterator over individual pixels in the image.
```

Comment:
This should also be added to the `tinybmp` docs.

IMO `Tga.into_iter()` could be confused with `Tga::into_iter()`.
```suggestion
//! To access the individual pixels in the images the `Tga` struct implements `IntoIterator`.
//! The example below demonstrates this feature by collect all pixels in a vector.
```